### DISCUSSION TITLE
Plotman creating mnt folder in Chia-Blockchain folder and plotting to it instead of non-OS temp directories

### DISCUSSION THREAD

**Discussion by DendyDarzartar** [ericaltendorf/plotman#811] [Q&A] [answered]
Hello,

When I run the plotman plot command a /mnt/ folder is created within the chia-blockchain folder on my OS drive and the drive begins to plot to it despite that drive not being listed as a temp directory within the config file. I had to give my user folder access in order to be able to delete the folder.

Does anyone know what could have caused this? I am wondering if I mistakenly partitioned and mounted the drives within a chia terminal window?

Everything looks correct when I check the /etc/fstab directory and the plotman's config.yaml yet attempting to plot will utilize one of the existing mount directories in name only but will plot it to the OS drive regardless of the fact that the temp directories appear to only point to the mnt folder and not to the chia-blochain directory

**Comment by altendky** [comment]
We will still need to at least see your configuration file.  Having the full paths to the directories you mention would be good as well.  Maybe the output of `ps aux | grep 'plots create' and `ps aux | grep plotman` as well.  And the output of however you are running plotman and monitoring it.

**Reply by DendyDarzartar** [reply]
Ok, this is about to sound very stupid but I figured it out with a little assistance.

It turns out the config file for plotman's temp directories did not have a "/" preceding the  "mnt/XXXX" value which was causing it to write to the chia-blockchain.

Thank you for following up, I will close this now and hopefully it might help someone else along the way.

**Reply by altendky** [reply]
I'm glad you got it figured out.  It seems like perhaps we should require that paths be absolute and not relative.  Sorry for the trouble.

**Comment by DendyDarzartar** [comment, marked as answer]
If you are reading this and have the same issue as described please verify that the plotman config.yaml file shows the tmp directory as "/mnt/XXXX" where XXXX is the name of the mounted directory if you just have "mnt/XXXX" without the forward slash to start you will have the same issue.

**Comment by DendyDarzartar** [comment]
No trouble at all! I appreciate you following up and I apologize for the
delays in getting back to you.
